### PR TITLE
Feat:added  home page

### PR DESCRIPTION
**Pull request by talrejah02**
Added home page
please review and give suggestions using this live [link](https://deploy-preview-1--playful-bombolone-e2e666.netlify.app/)

### CODE REVIEW

File: src/components/QuizCategory/CategoryCards.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    border-radius: 1px;
+    margin-bottom: 0.4rem;
+    color: var(--accent);
+    font-size: 1.2rem;
+    font-weight: 400;
+    transition: 0.4s;
+}
+.cart-btn:hover{
+    background-color:var(--accent) ;
+    color: #191A1A;
```
Comment (shuntzhere) [inline src/components/QuizCategory/CategoryCards.css:33]: Use variables for colors.

File: src/components/QuizCategory/CategoryCards.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./CategoryCards.css";
+import { NavLink } from "react-router-dom";
+
+function CategoryCards() {
+  return (
+    <section class="card shadow">
+      <img class="card-img" src="Images/cloud.jpg" alt="" />
```
Comment (shuntzhere) [inline src/components/QuizCategory/CategoryCards.jsx:8]: alt text missing

File: src/components/QuizCategory/CategoryCards.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      <section class="card-description">
+        <section class="card-detail">
+          <span class="card-title">product title</span>
+        </section>
+        <NavLink className="cart-btn" to="/Rules">
```
Comment (shuntzhere) [inline src/components/QuizCategory/CategoryCards.jsx:13]: the path should be /rules

**Review by shuntzhere — APPROVED** [review]
The link is working fine, code seems good too. Have left some suggestions, do consider.

**Review by sanathkumar26700 — APPROVED** [review]

**Review by ScoobyDubeyDo — APPROVED** [review]

**Pull request merged**